sky/utils/kubernetes/generate_kubeconfig.sh

```diff
@@ -169,6 +169,9 @@ roleRef:
   kind: Role
   name: skypilot-system-service-account-role
   apiGroup: rbac.authorization.k8s.io
+EOF
+# Apply optional ingress-related roles, but don't make the script fail if it fails
+kubectl apply -f - <<EOF || true
```

Comment:
Should we `echo` some message when it fails?

Reply:
thanks, added!